Equilobe.DailyReport.Models/Interfaces/IAtlassianWorklogDataService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Equilobe.DailyReport.Models.Dashboard;
+using Equilobe.DailyReport.Models.Storage;
+using System;
+using System.Collections.Generic;
+
+namespace Equilobe.DailyReport.Models.Interfaces
+{
+    public interface IAtlassianWorklogDataService : IService
+    {
+        List<InstanceWorklogs> GetAtlassianWorklogsByUserIds(long instanceId, List<long> users);
+        Dictionary<long, List<DashboardWorklog>> GetLastWorklogsByUsers(List<long> usersIds, ReportContext context, string baseUrl);
```

Comment:
Why don't you use here the same list as above?

Reply:
the method above is used to update stalling user, but this one is used for getting the data onto the dashboard. So it needs to work pretty fast. If I were to have a list, I'd have to search for the id that I want within each day of each member and search through a list that many times will drag the response time down. I'd rather save using that way above when it's used to update the db, and where time is not exactly of the essence.